testData/annotation.semantic/ErrorOnOptionalParameterWithNonConstMethod.hx

```diff
@@ -1,6 +1,6 @@
 package;
 
 class Test {
-  static inline var myConstant:Float = Std.int("12345".substr(2)) * 1; // Not really a constant.
+  static inline var myConstant:Float = Std.int(Std.parseFloat("12345".substr(2))) * 1;  // Not really a constant.
```

Comment:
because it does not compile.
![image](https://user-images.githubusercontent.com/3193925/102092238-2379ea00-3e20-11eb-8d8d-cd30b0dfed72.png)

```less
Source/Main.hx:11: characters 47-64 : String should be Float
Source/Main.hx:11: characters 47-64 : For function argument 'x'
Build halted with errors.
```